app/server.rb

```diff
@@ -453,22 +461,27 @@ def getChildDepts(unit)
   content_type :json
   id = "qt"+shortArk
   item = Item[id]
+  attrs = JSON.parse(Item.filter(:id => id).map(:attrs)[0])
   unitIDs = UnitItem.where(:item_id => id, :is_direct => true).order(:ordering_of_units).select_map(:unit_id)
   unit = unitIDs ? Unit[unitIDs[0]] : nil
 
   if !item.nil?
+    authors = ItemAuthors.filter(:item_id => id).order(:ordering).
+                 map(:attrs).collect{ |h| JSON.parse(h)}
+    citation = getCitation(shortArk, authors, attrs)
     begin
       body = {
         :id => shortArk,
+        :citation => citation,
+        :title => citation[:title],
+        # ToDo: Normalize author attributes across all components (i.e. 'family' vs. 'lname')
+        :authors => authors,
```

Comment:
I don't know that I want to modify our database to match the CDL-JSON schema; it would seem like overkill to me. And actually, you could put the translation code on the React side of things.